src/components/Comment/index.tsx

```diff
@@ -44,9 +85,46 @@ export class CommentBase extends React.Component<PublicProps> {
     }
   }
 
-  renderComment() {
-    const comment = this.props.comment as Comment;
+  onCommentChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    event.preventDefault();
 
+    this.setState({ commentText: event.target.value });
+  };
+
+  onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    const {
+      _manageComment,
+      addonId,
+      dispatch,
+      fileName,
+      initialComment,
+      line,
+      versionId,
+    } = this.props;
+    event.preventDefault();
```

Comment:
This doesn't seem to be covered by a test. Should it be?

Reply:
Personally I don't think it's such a useful thing to add a regression test for. When reading this code, it's clear that the intent is to prevent the default, so, why would a future developer change it? I also can't see how a refactor would accidentally remove this. 

Tests exist primarily to prevent accidental breakage by developers. Tests are not good at preventing *intentional* breakage 😅 

I see `event.preventDefault()` as something similar to putting `color="red"` in the JSX of a component. It's configuration. I'm open to other ideas about it, though.

As a counterpoint, let's say this method had to be called in an if statement, like:

```ts
onClick(event) {
  if (!this.props.allowDefault) {
    event.preventDefault();
  } 
}
```

In that case I'd want to see a regression test because it would be easy to accidentally break that in a refactor.